packages/dds/tree/src/simple-tree/proxies.ts

```diff
@@ -96,26 +102,14 @@ export function getProxyForField(field: FlexTreeField): TreeNode | TreeValue | u
 	}
 }
 
-/**
- * A symbol for storing TreeNodeSchema on FlexTreeNode's schema.
- */
-export const simpleSchemaSymbol: unique symbol = Symbol(`simpleSchema`);
```

Comment:
Note for reviewers: These were moved to `schemaCaching.ts` to prevent import cycles, and to remove the need to export the symbol.